syncd/FlexCounter.cpp

```diff
@@ -363,6 +370,7 @@ void deserializeAttr(
     sai_deserialize_acl_counter_attr(name, attr);
 }
 
+
 template <typename StatType>
 class CounterContext : public BaseCounterContext
```

Comment:
this should be in separate file 1 class per one file cpp

Reply:
This code is not added by me.